### DISCUSSION TITLE
Issue with selected Plan

### DISCUSSION THREAD

**Discussion by Bvpraagh** [opengisch/QField#3946] [Q&A]
Hi,
I have created an organization and this is the Pricing Plan I want. However, it seems that I have somehow been placed on the Pro Plan (It says Pro next to my name and the estimated invoice is for 9 Euro). My organization has been created.
Possibly my mistake somewhere along the line but I don't know how to fix it so I am on the Organization Plan? I particularly want to keep the discount. I applied yesterday.
Any suggestions?

thanks

**Comment by mojoblin** [comment]
Hi, 
Sharing the same issue here, I'd like to have the organization plan activated and be able to manage users in the organization, but stuck with "Pro" plan at the moment. Hope this could resolve once Beta end transistion phase is over.
Thanks
Marc

**Reply by Bvpraagh** [reply]
Thanks. So perhaps I didn't do anything wrong! Hopefully it will be resolved soon.

**Comment by lucienicolier** [comment]
Hi,
can you give me your organizations name?

I will have a look.

**Reply by mojoblin** [reply]
Hi @lucienicolier
Organization Kurrant
Users mojoblin and eraldo
Thanks a lot for your support
Marc-Olivier

**Comment by Bvpraagh** [comment]
HI, My Organisation name is INVERT-ECO. I have added one person to the organisation. I have added three screenshots below.

 

Thanks very much.

**Comment by lucienicolier** [comment]
Hi,

You need to go to your organization -> edit organization -> billing

There you can subscribe for the organization

**Reply by mojoblin** [reply]
Dear lucie, 

I Indeed found the Organization billing, but what remains unclear to me is why as a user (mojoblin) I'm subscribing to Pro plan with an estimated billing, while I own organization "Kurrant" which subscribes to Organization plan with 2 users (including mojoblin so myself as owner of the Organization) meaning that user mojoblin is, in theory and as displayed on the Qfield.cloud interface, billed twice : once as Pro user, once as member of the Organization. 
Am I missing something somewhere ?
Thanks for clarification

**Comment by Bvpraagh** [comment]
Thankyou. I can see that I have the correct information for the Organisation and subscription fee as I had already subscribed to organisation which has correct billing info. I think I accidentally subscribed to both plans.

 

If I delete this Pro Plan, will it then revert to only using my organisation plan? Its just when I go to cancel the Pro plan, it says it will revert to free plan.

 

Many Thanks,

Beverley

**Reply by lucienicolier** [reply]
If all your projects are in your organization, and you are a member of the organization, you are going to use the organization plan.

Best,

Lucie

**Comment by Bvpraagh** [comment]
This is exactly my situation too. I haven’t’ uploaded any projects to Organisation and I hope it is clear when I try to do this.  I am going to delete the Pro plan and hope it doesn’t affect anything.
